packages/core/src/button/icon-button.stories.ts

```diff
@@ -96,10 +96,10 @@ export function sizes() {
 export function block() {
   return html`
     <div cds-layout="vertical gap:sm align:horizontal-stretch">
-      <cds-icon-button block><cds-icon shape="user"></cds-icon></cds-icon-button>
-      <cds-icon-button block action="outline"><cds-icon shape="user"></cds-icon></cds-icon-button>
-      <cds-icon-button block size="sm"><cds-icon shape="user"></cds-icon></cds-icon-button>
-      <cds-icon-button block action="outline" size="sm"><cds-icon shape="user"></cds-icon></cds-icon-button>
+      <cds-icon-button><cds-icon shape="user"></cds-icon></cds-icon-button>
```

Comment:
Same here. What's happening with block buttons?

Reply:
Reverted the block attr removal then updated the demos to use block and not the layout stretch util